docs/probes.md

```diff
@@ -0,0 +1,43 @@
+# Liveness and Readiness Probes
+
+Liveness and readiness probes are a way for Kubernetes to constantly monitor your application and performing restarts or holding off traffic as needed. From the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/): 
```

Comment:
I may be wrong but I think 
"and performing restarts or holding off traffic as needed" should be 
"by performing restarts or holding off traffic as needed" or
"and perform restarts or hold off traffic as needed"